### DISCUSSION TITLE
〔 Comments Section 〕Fallout Shelter Online: Mysterious Treasure

### DISCUSSION THREAD

**Discussion by morgosus** [morgosus/morgosus.github.io#15] [Comments]
A comments section for the [Mysterious Treasure](http://www.toms.click/gaming/foso/2021/08/15/fallout-shelter-mysterious-treasure.html) post.

**Comment by morgosus** [comment]
If you're done with overseer's room, I suggest you focus on the quest hub to get wood :)

**Comment by Emptycz** [comment]
Does it matter which characters I assign in the repeatables?

**Reply by morgosus** [reply]
As far as I've observed, no. I tried them in various positions and the end sugar bomb gain was the same.

**Comment by morgosus** [comment]
Mysterious Treasure just became the main source of nameplates - which are required for the classroom dweller upgrades.
